### PR TITLE
Full draft  of user example complete

### PR DESCRIPTION
**Pull request by voremargot**
This is the first draft of the package examples for the documentation! Please take a look. 

@zackt113 I was going to put an example of how you can save your plots to your computer but I realized your function doesn't output the plot object, it only renders it! I am not sure if this is something we can easily change (ultimately it would be great if you could easily save your plots).

### CODE REVIEW

File: docs/example.ipynb [hunk 1]
```diff
@@ -2,30 +2,821 @@
  "cells": [
   {
    "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
    "source": [
-    "# Example usage\n",
+    "<center> <h1> How to use strapPy</h1> </center>\n",
```
Comment (gauthampughazhendhi) [inline docs/example.ipynb]:
```suggestion
    "<center> <h1> How to use StrapPy</h1> </center>\n",
```

@voremargot just a small name change

**Review by gauthampughazhendhi — CHANGES_REQUESTED** [review]
@voremargot this looks great! thanks for the good work. I have only one change

**Review by BooleanJulien — APPROVED** [review]
lgtm!

**Review by zackt113 — APPROVED** [review]
Everything looks good!

**Comment by voremargot** [comment]
Updates have been made! I will merge!

**Pull request merged**